### DISCUSSION TITLE
Double Preceision Coordinates Support Feature Request

### DISCUSSION THREAD

**Discussion by KuDeSnik33ra** [o3de/o3de#5253] [General]
If someone uses the Engine to create a huge scale world - for example, flight or railway simulator - 
double precision coordinates support is required. GPUs have limited double-precision performance,
consumer GPUs even have man-made limits for such performance.

The solution is to use double-precision transformation matrices for scene objects on the CPU side,
but translate all the objects to camera-space coordinates (to eliminate huge numbers), convert matrices
to single-precision and pass them to GPU in this form.

Is there any support for this or is it planned in the future?

**Comment by jesst3r** [comment]
Agreed
